resources/user-guides/restore_from_keys.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: static_page
+title: "Restoring a wallet from private keys"
+---
+
+
+
+Restoring a wallet from private keys (via the command line in this instance) is pretty simple. If you have the necessary information, with this guide you can completely restore your wallet. Note, you do NOT have to have your password to restore from keys.
```

Comment:
> Restoring a wallet from private keys (via the command line in this instance) is pretty simple.

I would drop the ' in this instance' because it really adds nothing and is more conversational than professional.

> with this guide

Capitalize.

> Note, you do NOT

Change to `Note: You do NOT` or `Note that you do NOT` here. Comma doesn't work because 'Note' is not a preposition. This sentence has two verbs: 'Note' and 'have'

>have to have

It's my personal taste, but I prefer `need to have`